CONTRIBUTING.md

```diff
@@ -17,28 +17,45 @@ The following is a set of guidelines for contributing to the UofT Coders communi
 
 ### About Uoft Coders
 
-The UofT Coders group was formed to share and learn about coding techniques and best practices for computing and analysis in research. We hold weekly sessions in the format of code-alongs, coworking sessions or journal clubs/discussions. To see our previous and upcoming events, you can visit our [Events repo](https://github.com/UofTCoders/Events/issues). To learn more about our group, you can read our [constitution](https://github.com/UofTCoders/council/blob/master/Constitution.md). 
+The UofT Coders group was formed to share and learn about coding techniques and best practices for computing and analysis in research. We hold weekly sessions in the format of code-alongs, coworking sessions or journal clubs/discussions. To see our previous and upcoming events, you can visit our [Events repo](https://github.com/UofTCoders/Events/issues). To learn more about our group, you can read our [constitution](https://github.com/UofTCoders/council/blob/master/Constitution.md).
 
 -----
-## How You Can Contribute
+# How You Can Contribute
 
-## Running a lesson
+## Running a Lesson
+### Lesson Instructor Checklist
+#### Making the Lesson:
 
-### Making the lesson
++ **Use Built-in Datasets**: Use built-in sample datasets instead of requiring attendees to download files.
++ **Keep in mind beginners**: make few assumptions about the knowledge of the audience, unless specified that this is an intermediate level lesson, requiring prior knowledge. Keep it simple.
++ **Minimal use of slides**: If your lesson involves coding, keep slides to a minimum if at all, focus on interactive live-coding.
++ **Code Review**: The lesson code is posted to GitHub along with a pull request to [UofTCoders/studyGroup](https://github.com/UofTCoders/studyGroup) repository at least 1 full day prior to the lesson date to allow for review. See [Making a Pull Request](#submitting-a-pull-request)
 
+
+#### Giving the Lesson
+
++ **Arrive early**: Come 10 minutes before the lesson starts to set up.
++ **Introduce yourself**: Start by introducing yourself and perhaps why you're teaching this lesson.
++ **Stay on time**: Keep mindful of the time, lessons are 50-60 minutes long.
++ **Start from the very beginning**. Actually show how to open the program or IDE (RStudio/Jupyter Notebook/Shell/etc)
+with everyone else (especially for beginner lessons).
++ **Live-coding**: Use of slides is minimal, majority of lesson involves writing the code WITH the audience during the lesson
++ **Stay on topic**: There is only one hour, if a question arises that is off-topic, you can always suggest discussing afterwards.
+-------
+## Lesson Tips
```

Comment:
This heading and all its contents can be removed. Your summary is better and there is no additional information here, just more words. The one exception is the introductory sentences about the Mozilla study group (line 54-59), move that to right above your new summary.